.pre-commit-config.yaml

```diff
@@ -77,7 +77,8 @@ repos:
         entry: pylint
         language: system
         types: [python]
-        args: ["-rn", "-sn", "--rcfile=pylintrc", "--fail-on=I"]
+        args:
+          ["-rn", "-sn", "--rcfile=pylintrc", "--fail-on=I", "--output-format=github"]
```

Comment:
I think this doesn't do much right? As `pre-commit` isn't uploaded to Github.

Instead we should use this in our workflows defined in `.github`

Reply:
Yes you are right, it is only needed for the manual stage hook which is run in the checks.yml workflow.